enterprise/apps.py

```diff
@@ -17,7 +17,7 @@ class EnterpriseConfig(AppConfig):
 
     name = "enterprise"
     valid_extensions = [".png", ]
-    image_size = 250 * 1024
+    image_size = 4 * 1024
```

Comment:
I kind of agree with @mattdrayer's initial assessment - 4kB is a bit small of a limit to not be configurable. I'd rather have this pull from the Django settings but have it default to 4kB.